### ISSUE TITLE
Unsupported `DEALLOCATE ALL`

### ISSUE THREAD

**Issue by morgothulhu** [oguimbal/pgsql-ast-parser#45]
Hello @oguimbal! Thanks for the continued support. Another small one. Thanks for supporting it in future.

```
DEALLOCATE ALL
```

ERROR:

```
DEALLOCATE ALL
with Error: Syntax error at line 1 col 12:

  DEALLOCATE ALL
             ^
Unexpected kw_all token: "all". I did not expect any more input. Here is the state of my parse table:

    kw_truncate → %word ●
    kw_delete → %word ●
    kw_start → %word ●
    kw_insert → %word ●
    kw_begin → %word ●
    kw_show → %word ●
    kw_set → %word ●
    kw_tablespace → %word ●
    kw_rollback → %word ●
    kw_commit → %word ●
    kw_comment → %word ●
    kw_raise → %word ●
    kw_drop → %word ●
    kw_alter → %word ●
    kw_update → %word ●
    kw_prepare → %word ●
```

**Comment by janpaepke** [comment]
Solution and discussion about AST output in https://github.com/oguimbal/pgsql-ast-parser/pull/74

**Comment by oguimbal** [comment]
Released in `pgsql-ast-parser@10.0.3` !
